pages/common/xh.md

```diff
@@ -0,0 +1,24 @@
+# xh
+
+> Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body (each key specified is added to the top-level JSON object e.g. `{"name": "john", "age": 25}`):
```

Comment:
Perhaps this will read better?
```suggestion
- Send a POST request with a JSON body (key-value pairs are added to a top-level JSON object - e.g. `{"name": "john", "age": 25}`):
```